KmsClient.js

```diff
@@ -185,23 +189,28 @@ export class KmsClient {
     if(!capability) {
       capability = `${ZCAP_ROOT_PREFIX}${encodeURIComponent(url)}`;
     }
+
     try {
-      // sign HTTP header
       const headers = await signCapabilityInvocation({
         url, method: 'post', headers: this.defaultHeaders,
         json: capabilityToRevoke, capability, invocationSigner,
         capabilityAction: 'write'
       });
+
       // send request
       const {agent} = this;
       await httpClient.post(url, {agent, headers, json: capabilityToRevoke});
     } catch(e) {
+      let errorMessage = 'Error revoking zCap.';
       if(e.status === 409) {
-        const err = new Error('Duplicate error.');
-        err.name = 'DuplicateError';
-        throw err;
+        e.name = 'DuplicateError';
```

Comment:
This `DuplicateError` needs the same adjustment as above.

Reply:
Refactored.